src/Components/CreateImageWizard/utilities/requestMapper.ts

```diff
@@ -558,6 +569,30 @@ const getOpenscap = (state: RootState): OpenScap | undefined => {
   return undefined;
 };
 
+const getUsers = (state: RootState): User[] | undefined => {
+  const users = selectUsers(state);
+
+  if (!users || users.length === 0) {
```

Comment:
It’s good to always to think really carefully about the types and their implications.

What’s the return type of selectUsers? It is <User>[].

So what does checking for `!users` do?

Well, users is either going to be an empty array or an array of <User> objects.

In both cases, `!users` returns false.

If you’re ever confused about what something will return, an easy way to check is using the developer console in your web browser:
![Screenshot from 2024-12-18 14-21-06](https://github.com/user-attachments/assets/5554d043-6aaa-478b-9007-fb7b69ae296c)